### ISSUE TITLE
How to visualize the first 10 or 20 words for all the topics

### ISSUE THREAD

**Issue by randomrdrand** [MaartenGr/BERTopic#257]
I was wondering if it's possible to visualize the first 10 or 20 words for all the topics in a user friendly way. So without printing the tf-idf scores just the topics in rows and the words as columns in a table for example

**Comment by MaartenGr** [comment]
For each topic you can extract its information with `topic_model.get_topic()`. This returns a list of tuples with the word and corresponding c-TF-IDF value. You can iterate over those and only extract the words and skip over the value.

**Comment by MaartenGr** [comment]
After training the model, you play around with something like this:

```python
{topic: [word for word, ctfidf in topic_model.get_topic(topic)] for topic in set(topics)}
```

How you approach putting them in a table-like format is up to you.

**Comment by MaartenGr** [comment]
Yes, you can set `top_n_words` in BERTopic as described [here](https://maartengr.github.io/BERTopic/api/bertopic.html#bertopic._bertopic.BERTopic.__init__). However, the top 10 give the best representation of the topics whilst increasing this value gives very little additional information that isn't already contained in the top 10. You can see how little effect is has when adding words to a topic representation by running `topic_model.visualize_term_rank()`.
